### PR TITLE
Fixup for RFC, documentation and feature request templates

### PR DESCRIPTION
**Pull request by jasukhar**
Fixed broken meta-data for github templates.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/RFC.md [hunk 1]
```diff
@@ -1,7 +1,6 @@
 ---
 name: RFC for new interface
-about: Use this template to request new functionality or change behavior of the
-library
+about: Use this template to request new functionality or change behavior of the library
```
Comment (vmalia) [inline .github/ISSUE_TEMPLATE/RFC.md:3]: "change **the** behavior of the library"?

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -1,7 +1,6 @@
 ---
 name: Request a feature
-about: Use this template to request new functionality or change behavior of the
-library
+about: Use this template to request new functionality or change behavior of the library
```
Comment (vmalia) [inline .github/ISSUE_TEMPLATE/feature_request.md:3]: "change **the** behavior of the library"?

File: .github/ISSUE_TEMPLATE/RFC.md (hunk 1 above)
Comment (jasukhar) [reply .github/ISSUE_TEMPLATE/RFC.md]: fixed

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 2 above)
Comment (jasukhar) [reply .github/ISSUE_TEMPLATE/feature_request.md]: fixed

**Pull request merged**